crates/protocol/proto/str/packet/v1/packet.proto

```diff
@@ -5,16 +5,16 @@ package str.packet.v1;
 import "str/gossip/v1/gossip.proto";
 import "str/messages/v1/messages.proto";
 import "str/crypto/v1/crypto.proto";
+import "str/sendavailablepeers/v1/sendavailablepeers.proto";
+import "str/requestavailablepeers/v1/requestavailablepeers.proto";
```

Comment:
Should be a unified module, `peers.proto` perhaps.

Reply:
another good idea.